mockgcp/mockcontainer/lro.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mockcontainer
+
+import (
+	"context"
+	"strings"
+
+	pb "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/generated/mockgcp/container/v1beta1"
+	lropb "google.golang.org/genproto/googleapis/longrunning"
+)
+
+func (s *ClusterManagerV1) newLRO(ctx context.Context, projectID string) (*pb.Operation, error) {
+	lro, err := s.operations.NewLRO(ctx)
+	if err != nil {
+		return nil, err
+	}
+	selfLinkPrefix := "https://container.googleapis.com/v1beta1/projects/" + projectID + "/global/operations/"
```

Comment:
Should we be using the v1beta1 API here? I thought GKE was pushing for everyone to use the V1 container.googleapi?

Reply:
Our terraform provider is using v1beta1, so this is what we have to mock today.

We could also support v1 (and we'll likely have to eventually), but I figured we should start with the codepath we're using today.